docs/user/system_upgrades.md

```diff
@@ -0,0 +1,98 @@
+# Upgrading your system
+
+Brewblox is designed to be compatible with the majority of Linux-based systems.
+To achieve this, it has two major dependencies: Docker, and Python.
+Both are available on most operating systems, but released versions have an End of Life date.
+Beyond this date, they are no longer supported, and compatibility is no longer guaranteed.
+
+For us, that means that after an End of Life date, tooling or dependencies may suddenly break.
+This can be errors during development, or during installation, but the end result is the same:
```

Comment:
This can be errors?

Shorten to: This means that we cannot guarantee an error-free experience on a system with outdated dependencies.